src/tests/registration/Registration.int.test.tsx

```diff
@@ -201,61 +214,66 @@ describe("RegistrationForm", () => {
-        email: TestValue.Email,
-        guests: [
-          {
-            date_of_birth: moment(TestValue.GuestDob, "MMDDYYYY").format(
-              "YYYY-MM-DD"
-            ),
-            first_name: TestValue.GuestFirstName,
-            information: {
-              [TEST_GUEST_DYNAMIC_FIELD.id]: TestValue.GuestFavouriteColour,
-            },
-            last_name: TestValue.LastName,
+    await waitFor(() => expect(FamilyAPI.postFamily).toHaveBeenCalledTimes(1));
+    expect(FamilyAPI.postFamily).toHaveBeenCalledWith({
+      address: TestValue.Address,
+      cell_number: TestValue.CellNumber,
+      children: [
+        {
+          date_of_birth: moment(TestValue.ChildDob, "MMDDYYYY").format(
+            "YYYY-MM-DD"
+          ),
+          first_name: TestValue.ChildFirstName,
+          information: {
+            [TEST_CHILD_DYNAMIC_FIELD.id]: TestValue.ChildFavouriteColour,
           },
-        ],
-        home_number: TestValue.HomeNumber,
-        interactions: [],
-        notes: TestValue.Notes,
-        parent: {
-          date_of_birth: moment(TestValue.ParentDob, "MMDDYYYY").format(
+          last_name: TestValue.LastName,
+        },
+      ],
+      email: TestValue.Email,
+      guests: [
+        {
+          date_of_birth: moment(TestValue.GuestDob, "MMDDYYYY").format(
             "YYYY-MM-DD"
           ),
-          first_name: TestValue.ParentFirstName,
+          first_name: TestValue.GuestFirstName,
           information: {
-            [TEST_PARENT_DYNAMIC_FIELD.id]: TestValue.ParentFavouriteColour,
-            [TEST_SESSION_DYNAMIC_FIELD.id]: TestValue.TimeInCanada,
+            [TEST_GUEST_DYNAMIC_FIELD.id]: TestValue.GuestFavouriteColour,
           },
           last_name: TestValue.LastName,
         },
-        preferred_comms: "",
-        preferred_number: "",
-        work_number: TestValue.WorkNumber,
+      ],
+      home_number: TestValue.HomeNumber,
+      interactions: [],
+      notes: TestValue.Notes,
+      parent: {
+        date_of_birth: moment(TestValue.ParentDob, "MMDDYYYY").format(
+          "YYYY-MM-DD"
+        ),
+        first_name: TestValue.ParentFirstName,
+        information: {
+          [TEST_PARENT_DYNAMIC_FIELD.id]: TestValue.ParentFavouriteColour,
+          [TEST_SESSION_DYNAMIC_FIELD.id]: TestValue.TimeInCanada,
+        },
+        last_name: TestValue.LastName,
       },
+      preferred_comms: "",
+      preferred_number: "",
+      work_number: TestValue.WorkNumber,
+    });
+
+    await waitFor(() =>
+      expect(EnrolmentAPI.postEnrolment).toHaveBeenCalledTimes(1)
+    );
+    expect(EnrolmentAPI.postEnrolment).toHaveBeenCalledWith({
+      enrolled_class: null,
+      family: 1,
       preferred_class: session.classes[1].id,
       session: session.id,
       status: EnrolmentStatus.SIGNED_UP,
+      students: [2, 3, 4],
     });
 
-    expect(onSubmit).toHaveBeenCalledTimes(1);
-  });
+    expect(onRegister).toHaveBeenCalledTimes(1);
+  }, 15000);
```

Comment:
I'm guessing this is some sort of timeout for the describe call?

Reply:
yeah it's a long test 😔 had to bump it up from 5 seconds